src/facebook_example/bot.clj

```diff
@@ -28,6 +29,17 @@
       ; If no rules apply echo the user's message-text input
       :else (reaction/echo message-text))))
 
+(defn on-quick-reply [event]
```

Comment:
We probably need a comment explaining `called by handle-message when the user has tapped a quick reply. <link-to-quick-rely-docs-on-fb-developer>`